project-koku/koku
# COST-748: Create cost materialized views for gcp.

I created the cost materialized views for GCP.

# How to test

Bring up the django server, and run `make gcp-source` make target. After the data is downloaded, process, and summarized check the materialized view tables for GCP and make sure they were populated correctly.
